scripts/install-chef-suse.sh

```diff
@@ -42,6 +42,7 @@ while test $# -gt 0; do
         -h|--help|--usage|-\?) usage ;;
         -v|--verbose) CROWBAR_VERBOSE=1 ;;
         -d|--debug) CROWBAR_DEBUG=1 ;;
+        -c|--crowbar) CROWBAR_FIRST_INSTALL=1 && CROWBAR_VERBOSE=1 ;;
```

Comment:
s/CROWBAR_FIRST_INSTALL/CROWBAR_WIZARD_MODE/ ? Or something like this. It's not necessarily a first install :-)